securedrop/journalist_templates/login.html

```diff
@@ -3,12 +3,21 @@
 
 <h1>{{ gettext('Login to access the journalist interface') }}</h1>
 
+<noscript>
```

Comment:
You should add this to the `sass` and have the default be `display: none`. The JS can set `display: block` or whatever to enable it when the page loads.

Reply:
That makes sense. I'll make the changes. Thanks @heartsucker